tests/Operation/WatchFunctionalTest.php

```diff
@@ -1370,11 +1370,11 @@ public function testOriginalReadPreferenceIsPreservedOnResume(): void
         }
 
         $changeStream = $operation->execute($secondary);
-        $previousCursorId = $changeStream->getCursorId(true);
+        $previousCursorId = $changeStream->getCursorId();
         $this->forceChangeStreamResume();
 
         $changeStream->next();
-        $this->assertNotSame($previousCursorId, $changeStream->getCursorId(true));
+        $this->assertNotSame($previousCursorId, $changeStream->getCursorId());
```

Comment:
I don't think `assertNotSame()` was ever correct here, as it previously would have been comparing two CursorId objects. I suggest switching this to `assertNotEquals()`.

Also, please double check any other cursor ID comparisons if it's not too much trouble.

The original line here seems to date back to https://github.com/mongodb/mongo-php-library/commit/aac3c6bd84ed39708c156d72dcfde7e97212254b

Reply:
Ah, excellent point. I've looked through all usages of `CursorInterface::getId` and `ChangeStream::getCursorId` and confirmed that they only use `assertNotEquals` as well as loose comparisons (`==`) throughout. 👍